Makefile

```diff
@@ -1,4 +1,4 @@
-OCAMLBUILD=ocamlbuild
+OCAMLBUILD=ocamlbuild -cflag -g
```

Comment:
ups, that was not meant to be in there. I still wonder if ocamldebug is really hideous or if I don't know how to use it.